core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml

```
@@ -3,6 +3,8 @@ tide:
   - labels:
     - approved
     - lgtm
+    - jira/valid-bug
+    - jira/valid-reference
```

Hey, as said on slack, this adds the requirement that every PR contains both labels which isn't the case for features.

If you want that the `tide` job passes with one of both labels you need two queries as we use in the [openshift/console/_prowconfig.yaml](https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/console/_prowconfig.yaml)

It's up to you if you accept valid-bug + valid-reference for bugs and then only valid-reference for features in two queries. Or you remove the valid-bug for now until features and bugs has other requirments.

```suggestion
    - jira/valid-reference
```

---

updated it.